packages/menu/src/MenuItem/MenuItem.tsx

```diff
@@ -52,10 +57,18 @@ export const MenuItem = InferredPolymorphic<MenuItemProps, 'button'>(
       variant = Variant.Default,
       ...rest
     },
-    ref: React.Ref<any>,
+    fwdRef: React.Ref<any>,
   ) => {
     const { Component } = useInferredPolymorphic(as, rest, 'button');
-    const { theme } = useContext(MenuContext);
+    const { theme, highlightIndex: _highlightIndex } = useContext(MenuContext);
+    const { index: _index, ref } = useDescendant(
```

Comment:
same question on the `_*` variables

Reply:
ditto. Will be used in https://github.com/mongodb/leafygreen-ui/pull/2369